### DISCUSSION TITLE
How to lookup another field from different object and get values?

### DISCUSSION THREAD

**Discussion by Abdalmohsen-H** [pimcore/pimcore#14195] [Q&A / Support]
Hi , I am moving from spreadsheets based system to pimcore
If I have tow data objects both have SKU field
I want to know how to do lookup from another data object based on identifier for example SKU for matching
Then get this field is value that match the same SKU from Pimcore GUI dashboard

I want to know what are the possible methods to do this lookup from  Pimcore GUI dashboard also how to do formulas (calculations or logic).

P.S. I am python developer If coding this in PHP is the only way I could learn with a given example
Thanks

**Comment by frithjof** [comment]
take a look at the php api for data objects ... also extending data object classes for calculations
